addons/l10n_ch/models/account_invoice.py

```diff
@@ -41,7 +41,8 @@ class AccountMove(models.Model):
     def _compute_l10n_ch_qr_is_valid(self):
         for move in self:
             move.l10n_ch_is_qr_valid = move.move_type == 'out_invoice' \
-                                       and move.partner_bank_id._eligible_for_qr_code('ch_qr', move.partner_id, move.currency_id, raises_error=False)
+                                       and move.partner_bank_id._eligible_for_qr_code('ch_qr', move.partner_id, move.currency_id, raises_error=False) \
```

Comment:
One more reason why :weary: 
https://peps.python.org/pep-0008/#maximum-line-length
> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Reply:
The line was already using backslashes, so I continued... I will change this